packages/PHPStanStaticTypeMapper/Contract/TypeMapperInterface.php

```diff
@@ -12,6 +12,9 @@
 use PHPStan\Type\Type;
 use Rector\PHPStanStaticTypeMapper\ValueObject\TypeKind;
 
+/**
+ * @template T of Type
+ */
 interface TypeMapperInterface
```

Comment:
should this interface be marked as internal, because we change it at will without considering BC in minor versions?

```suggestion
/**
 * @template T of Type
 * 
 * @internal
 */
interface TypeMapperInterface
```

Reply:
We don't use such convention, so no need to.